web/src/styles/scale.ts

```diff
@@ -0,0 +1,26 @@
+import tokens from '../tokens/tokens.json'
+
+const {
```

Comment:
I think I'll use this in the `Space` component instead of a string value. Thanks!